### PR TITLE
docs: make use of `info.releaseNotesURL`

### PR DESCRIPTION
**Pull request by ckipp01**
This just ensures that when Scala Steward sends in a PR to your project
using mainargs that it correctly points to the changelog allowing people
to easily just click it and go directly to it.

### CODE REVIEW

**Review by lefou — COMMENTED** [review]
Looks good to me. Since you defined some Github settings like `githubOrg`, `githubRepo` and th url in their own `val`s, it would be nice to use them consistently in all other `pomSettimgs` properties.

**Comment by ckipp01** [comment]
> Looks good to me. Since you defined some Github settings like `githubOrg`, `githubRepo` and th url in their own `val`s, it would be nice to use them consistently in all other `pomSettimgs` properties.

Ah, good call. Oversight on my part. Fixed.

File: build.sc [hunk 1]
```diff
@@ -27,12 +32,16 @@ trait MainArgsPublishModule
...
 
+  def publishProperties = super.publishProperties() ++ Map(
+    "info.releaseNotesURL" -> changelogUrl
+  )
+
   def pomSettings = PomSettings(
     description = "Main method argument parser for Scala",
-    organization = "com.lihaoyi",
-    url = "https://github.com/com-lihaoyi/mainargs",
+    organization = githubOrg,
```
Comment (lefou) [inline build.sc:41]:
```suggestion
    organization = "com.lihaoyi",
```

**Review by lefou — COMMENTED** [review]
Thank you. Found one little but important discrepancy. The maven org isn't the same as the github org.

File: build.sc (hunk 1 above)
Comment (ckipp01) [reply build.sc]: Ah good catch.

**Pull request merged**